docs/UMLdiagrams/model.puml

```diff
@@ -47,24 +59,23 @@ class Order {
 + getSize() : int
 + getTotalPrice() : double
 + getReceipt() : String
++ getRestaurantName() : String
++ getRestaurantAddress() : String
++ getUserName() : String
++ getOrderType() : String
++ getOrderItemList() : ArrayList<MenuItem>
 + getOrderSummary() : String
 + toString() : String
 }
-enum SetMenu {
-    Breakfast
-    Lunch
-    Dinner
-}
 
 Item <|-- MenuItem
 
 ItemManager <|.. Menu
 ItemManager <|.. Order
 
-MenuItem "1" <..  Order : order
-MenuItem "1" <.. Menu : menu
+MenuItem "1..*" -- "0..*" Order
+MenuItem "1..*" -* Menu
```

Comment:
Order should contain menuItem, but MenuItem should not contain order. Something is wrong.

Reply:
Good point, I have updated it to be a composition instead.